src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java

```diff
@@ -138,7 +147,7 @@ public void deleteRoles(final Node node) throws RepositoryException {
         // Read the effective Acl and return a list of acl:Authorization statements
         final List<WebACAuthorization> authorizations = effectiveAcl
                 .map(uncheck(x -> getAuthorizations(x.getLeft().toString())))
-                .orElse(new ArrayList<>());
+                .orElseGet(() -> getDefaultAuthorizations());
```

Comment:
Is it correct then, that the algorithm of determining the effective policy is now different than documented here:
https://wiki.duraspace.org/display/FEDORA4x/WebAC+Authorization+Delegate#WebACAuthorizationDelegate-Stepsindeterminingtheeffectiveauthorization

The default is not "Deny Access", but rather, "whatever is in the root-authorization.rdf"?